packages/loader/container-utils/src/error.ts

```diff
@@ -165,21 +166,18 @@ export class DataProcessingError extends LoggingError implements IErrorBase, IFl
 
         const normalizedError = normalizeError(originalError, { props });
 
-        // Check for errors that originated externally to our code before being normalized.
```

Comment:
This diff looks crazy.  All I did was update the if statement to use the new `isExternalError` helper, and invert it to do early return and reduce nesting